docs/learn/expert-guides/key-manager/execute-relay-transactions.md

```diff
@@ -212,358 +231,291 @@ const abiPayload = universalProfile.methods
   [
     // MUST be number `25`
-    LSP25_VERSION,       // `0x0000000000000000000000000000000000000000000000000000000000000019`
+    // Encoded value: `0x0000000000000000000000000000000000000000000000000000000000000019`
+    LSP25_VERSION,    
+
     // e.g: `4201` for LUKSO Testnet
-    chainId,             // `0x0000000000000000000000000000000000000000000000000000000000001069`
-    // e.g: nonce number 5 of the signer key X 
-    // (the private key associated with the address of the controller that want to execute the payload)
-    nonce,              // `0x0000000000000000000000000000000000000000000000000000000000000005`
+    // Encoded value: `0x0000000000000000000000000000000000000000000000000000000000001069`
+    chainId,      
+
+    // e.g: nonce number 5 of the signing controller that wants to execute the payload
+    // Encoded value: `0x0000000000000000000000000000000000000000000000000000000000000005`
+    nonce,          
+                      
     // e.g: valid until 1st January 2025 at midnight (GMT).
     // Timestamp = 1735689600
-    validityTimestamps, // `0x0000000000000000000000000000000000000000000000000000000067748580`
+    // Encoded value: `0x0000000000000000000000000000000000000000000000000000000067748580`
+    validityTimestamps,      
+
     // e.g: not funding the contract with any LYX (0)
-    msgValue,           // `0x0000000000000000000000000000000000000000000000000000000000000000`
-    // e.g: execute(uint256,address,uint256,bytes) -> send 3 LYX to address `0xcafecafecafecafeafecafecafecafeafecafecafecafeafecafecafecafe`
-    abiPayload,         // `0x44c028fe0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000cafecafecafecafecafecafecafecafecafecafe00000000000000000000000000000000000000000000000029a2241af62c000000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000000`
+    // Encoded value: `0x0000000000000000000000000000000000000000000000000000000000000000`
+    msgValue,    
+           
+    // e.g: send 3 LYX to address 0xcafecafecafecafecafecafecafecafecafecafe
+    // by calling execute(uint256,address,uint256,bytes)
+    // Encoded value: `0x44c028fe00000000000000000000000000000000000000000000000000000000        
+    //                 00000000000000000000000000000000cafecafecafecafecafecafecafecafeca 
+    //                 fecafecafecafe00000000000000000000000000000000000000000000000029a2 
+    //                 241af62c0000000000000000000000000000000000000000000000000000000000
+    //                 000000008000000000000000000000000000000000000000000000000000000000
+    //                 00000000`
+    abiPayload,         
   ],
 );
 
-let eip191Signer = new EIP191Signer();
+// Instanciate EIP191 Signer
+const eip191Signer = new EIP191Signer();
 
-let { signature } = await eip191Signer.signDataWithIntendedValidator(
+/**
```

Comment:
Suggest removing this code comment block for brevity. The code and function parameters are self explanatory themselves.